shims/spark304/src/main/scala/com/nvidia/spark/rapids/shims/spark304/RapidsShuffleInternalManager.scala

```diff
@@ -17,8 +17,9 @@
 package org.apache.spark.sql.rapids.shims.spark304
 
 import org.apache.spark.{SparkConf, TaskContext}
+
```

Comment:
style issue

Reply:
this still looks like issue?

Reply:
oh this is because https://github.com/NVIDIA/spark-rapids/issues/3388# 
I actually fixed these in https://github.com/NVIDIA/spark-rapids/pull/3411 so I guess its ok here